### DISCUSSION TITLE
Swift Doc

### DISCUSSION THREAD

**Discussion by trstn125** [supabase/supabase#27907] [Questions] [answered]
Hello,
How is it possible to download the whole .md documentation of Swift? I can not find on the git.
I am referring to : https://supabase.com/docs/reference/swift/
Thank you so much

**Comment by GaryAustin1** [comment, marked as answer]
I just searched on a couple of key words that seemed "swifty" in supabase/supabase repository where the docs are...
https://github.com/supabase/supabase/blob/a69e67b063746cbe830e6b69f0d5be24b6b451c6/apps/docs/spec/supabase_swift_v2.yml#L2376
Is that what you are looking for?
